Docs/how_to_build_on_linux.md

```diff
@@ -1,136 +1,194 @@
-and build the different modules
+<h4>Get assets</h4>
 
+Only the assets package, the visual content, is yet to be donwloaded. These are stored separately to make the repository a bit lighter. CARLA cannot be built without the assets, so there is a script that downloads and extracts the latest content version (this package is >3GB, it might take some time depending on internet connection).  
+Get into the root carla folder. The path should correspond with the repository just cloned:
 ```sh
-make launch     # Compiles the simulator and launches Unreal Engine's Editor.
-make PythonAPI  # Compiles the PythonAPI module necessary for running the Python examples.
-make package    # Compiles everything and creates a packaged version able to run without UE4 editor.
-make help       # Print all available commands.
+cd ~/carla
 ```
-
-Updating CARLA
---------------
----
-
-Every new release of CARLA, we release too a new package with the latest changes
-in the CARLA assets. To download the latest version and recompile CARLA, run
-
+Run the script to get the assets:
 ```sh
-make clean
-git pull
 ./Update.sh
-make launch
 ```
 
-- - -
-
-Assets repository (development only)
-------------------------------------
 
-Our 3D assets, models, and maps have also a
-[publicly available git repository][contentrepolink]. We regularly push latest
-updates to this repository. However, using this version of the content is only
-recommended to developers, as we often have work in progress maps and models.
+<h4>Set the environment variable </h4>
 
-Handling this repository requires [git-lfs][gitlfslink] installed in your
-machine. Clone this repository to "Unreal/CarlaUE4/Content/Carla"
+For CARLA to find the Unreal Engine 4.22 installation folder, an environment variable needs to be set.
 
 ```sh
-git lfs clone https://bitbucket.org/carla-simulator/carla-content Unreal/CarlaUE4/Content/Carla
+export UE4_ROOT=~/UnrealEngine_4.22
 ```
 
-It is recommended to clone with "git lfs clone" as this is significantly faster
-in older versions of git.
+This variable should be added to `~/.bashrc` or `~/.profile` to set it persistently session-wide. If not, it will only be accessible for current shell.
+
+<h4>make CARLA</h4>
+
+The last step is to finally build CARLA. There are different `make` commands to build the different modules. All of them run in the root CARLA folder: 
+
+* **launch** will compile the server simulator and launch it in Unreal Engine. Press **Play** to start the spectator view. Camera can be moved with WASD keys and rotated by clicking the scene while moving the mouse around. Close the editor window to exit:  
+```sh
+make launch
+``` 
+* **PythonAPI** will compile the API client, necessary to grant control over the simulation. After building it, scripts can run. The following example will run a dynamic weather on the scene: 
+```sh
+make PythonAPI && cd PythonAPI/examples && ./dynamic_weather.py
```

Comment:
Maybe a more visual example related to the cars? like spawn_npc.py?

Reply:
@glopezdiest Updated the file with corresponding suggestions in mind.